Kbs.Wpf/Reservation/Create/SelectBoatType/SelectBoatTypePage.xaml.cs

```diff
@@ -28,12 +30,13 @@ public SelectBoatTypePage(INavigationManager navigationManager)
         {
             ViewModel.Items.Add(new SelectBoatTypeBoatTypeViewModel(type));
         }
+        
     }
 
     public void BoatTypeSelected(object sender, MouseButtonEventArgs e)
     {
-        UserRole waarde = SessionManager.Instance.Current.User.Role;
-        if (waarde == UserRole.Member)
+
+        if (User.Role == UserRole.Member)
```

Comment:
Verander naar: User.IsMember():
```suggestion
        if (User.IsMember())
```